spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbClientAutoConfiguration.java

```diff
@@ -30,102 +29,107 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.autoconfigure.condition.NoneNestedConditions;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.boot.context.properties.PropertyMapper;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Conditional;
 import org.springframework.context.annotation.Configuration;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
-import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
 import software.amazon.awssdk.regions.providers.AwsRegionProvider;
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClientBuilder;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
+import software.amazon.dax.ClusterDaxAsyncClient;
 import software.amazon.dax.ClusterDaxClient;
 
 /**
- * {@link EnableAutoConfiguration Auto-configuration} for DynamoDB integration.
+ * {@link DynamoDbClientAutoConfiguration Auto-configuration} for DynamoDB integration.
  *
  * @author Matej Nedic
  * @author Arun Patra
  * @since 3.0.0
  */
 @AutoConfiguration
-@EnableConfigurationProperties(DynamoDbProperties.class)
-@ConditionalOnClass({ DynamoDbClient.class, DynamoDbEnhancedClient.class, DynamoDbTemplate.class })
```

Comment:
`@ConditionalOnClass({DynamoDbClient.class}` 
Should be left.

Reply:
But it's there